docs/en/reference/transactions.rst

```diff
@@ -50,7 +50,7 @@ constants:
 
 The default transaction isolation level of a
 ``Doctrine\DBAL\Connection`` is chosen by the underlying platform
-but it is always at least READ\_COMMITTED.
+but it is always at least READ_COMMITTED.
```

Comment:
We can make it look even better:
```suggestion
but it is always at least ``READ_COMMITTED``.
```

By the way, why are you contributing this to `master`? Are other branches unaffected?

Reply:
Thanks! All branches are affected so where I should contribute? To each branch separately?

Reply:
You should contribute to the default branch: 2.10.x, we will then merge this up. `git rebase --onto 2.10.x master fix-transactions-doc` is your friend :)

Reply:
Done, thank you!